templates/CRM/Event/Form/Registration/Register.tpl

```diff
@@ -168,6 +167,30 @@
   {/if}
   {literal}
 
+  function additionalParticipantsChange() {
+    toggleAdditionalParticipants();
+    allowParticipant();
+  }
+
+  function toggleAdditionalParticipants() {
+    var submit_button = cj("#crm-submit-buttons > button").html();
+  {/literal}
+    var review_translated = '{ts}Review{/ts}';
+    var continue_translated = '{ts}Continue{/ts}';
+  {literal}
+    if (cj('#additional_participants').val()) {
+      cj("#additionalParticipantsDescription").show();
+      cj("#crm-submit-buttons > button").html(submit_button.replace(review_translated, continue_translated));
+    } else {
+      cj("#additionalParticipantsDescription").hide();
+      cj("#crm-submit-buttons > button").html(submit_button.replace(continue_translated, review_translated));
+    }
+  }
+
+  window.onload = function() {
+    toggleAdditionalParticipants();
+  };
```

Comment:
Not your fault, but the javascript style in this `.tpl` file is horribly outdated. You don't have to clean up the existing code (although if you feel inspired to do so, by all means open another PR for that) but please use the new style for new code.
- Enclose your code with `CRM.$(function($) {...});` to avoid creating global functions.
- Use `$` instead of the deprecated `$` within the closure.
- Keep the whole thing literal, only break out of literal within an actual string and use escaping, e.g. `var review_translated = '{/literal}{ts escape="js"}Review{/ts}{literal}';
- You don't need to use `window.onload` as the closure automatically waits for `$(document).ready()`

Reply:
Better late than never here, but this is now done. Thanks.